data/chatchannels/scripts/advertising.lua

```diff
@@ -16,8 +16,8 @@ function onSpeak(player, type, message)
 		return true
 	end
 
-	if player:getLevel() == 1 then
-		player:sendCancelMessage("You may not speak into channels as long as you are on level 1.")
+	if player:getLevel() < 20 or not player:isPremium() then
```

Comment:
```suggestion
	if player:getLevel() < 20 and not player:isPremium() then
```
That logic seems to be not what was intended.